invite/settings.py

```diff
@@ -36,4 +36,4 @@ def get_service_name():
     return getattr(
         settings,
         'INVITE_SERVICE_NAME',
-        Site.objects.get_current().name)
+        Site.objects.get_current(request=request).name)
```

Comment:
And considering that the current version of Django is 1.10.6, I think making that compatibility change is reasonable.